geomesa-accumulo/geomesa-accumulo-datastore/src/main/scala/org/locationtech/geomesa/accumulo/data/stats/GeoMesaMetadataStats.scala

```diff
@@ -207,62 +218,67 @@ class GeoMesaMetadataStats(val ds: AccumuloDataStore, statsTable: String)
     * @param merge merge with the existing stat - otherwise overwrite
     */
   private [stats] def writeStat(stat: Stat, sft: SimpleFeatureType, merge: Boolean): Unit = {
+
+    def shouldWrite(keyAndStat: (String, Stat)): Boolean = {
+      if (merge && keyAndStat._2.isInstanceOf[CountStat]) {
```

Comment:
[picky] Is it reasonable to create a case class for ```keyAndStat``` to avoid the ```._2``` reference (and the similar references elsewhere within this method)?